tests/pipeline/test_us_equity_pricing_loader.py

```diff
@@ -175,43 +175,43 @@
     [
         # Before query range, should be excluded.
         {'declared_date': Timestamp('2015-05-01', tz='UTC').to_datetime64(),
-         'ex_date': Timestamp('2015-06-01', tz='UTC').to_datetime64(),
+         'ex_date': Timestamp('2015-06-02', tz='UTC').to_datetime64(),
          'record_date': Timestamp('2015-06-03', tz='UTC').to_datetime64(),
          'pay_date': Timestamp('2015-06-05', tz='UTC').to_datetime64(),
          'amount': 90.0,
          'sid': 1},
         # First day of query range, should be excluded.
         {'declared_date': Timestamp('2015-06-01', tz='UTC').to_datetime64(),
-         'ex_date': Timestamp('2015-06-10', tz='UTC').to_datetime64(),
+         'ex_date': Timestamp('2015-06-11', tz='UTC').to_datetime64(),
          'record_date': Timestamp('2015-06-15', tz='UTC').to_datetime64(),
          'pay_date': Timestamp('2015-06-17', tz='UTC').to_datetime64(),
          'amount': 80.0,
          'sid': 3},
         # Third day of query range, should have last_row of 2
         {'declared_date': Timestamp('2015-06-01', tz='UTC').to_datetime64(),
-         'ex_date': Timestamp('2015-06-12', tz='UTC').to_datetime64(),
+         'ex_date': Timestamp('2015-06-15', tz='UTC').to_datetime64(),
```

Comment:
We currently ignore `record_date` completely, which may be a deficiency. But will fix so that it is more accurate for if/when we do use `record_dates`.